operator/charts/paladin-operator/values.yaml

```diff
@@ -8,6 +8,10 @@ mode: devnet
 # Number of Paladin and Besu nodes to deploy. Relevant only when mode is set to "devnet".
 nodeCount: 3
 
+# Number of Paladin nodes to deploy. When not set, defaults to nodeCount.
```

Comment:
I wonder if this would be clearer if it was `besuNodeCount`, which defaults to `null` i.e. "same number of Besus as Paladins`, but when set to e.g. `1` then you'd get `nodeCount` Paladins and 1 Besu

Reply:
[Here is how](https://github.com/LF-Decentralized-Trust-labs/paladin/pull/758/commits/23a8fbe8619f83e1453c9fd10e81a4adeda951f6#diff-3c051958b99c539b089e90f60189a7d11a1d8c307d3e77f62b7a865ce82f7b4bR5) I ended up working around the number of besu nodes